GameState.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2021 Lightricks. All rights reserved.
+
+class GameState:
+
+    MODE_DICT = {
```

Comment:
As the comment above - this probably was meant to be an enumeration, unless you have a reason and then let's discuss it.